src/Windowing/Silk.NET.Windowing.Desktop/GlfwWindow.cs

```diff
@@ -746,13 +772,28 @@ private unsafe void InitializeCallbacks()
             _glfw.SetDropCallback(_windowPtr, _onFileDrop);
         }
 
+        /// <summary>
+        /// Checks whether or not Vulkan is supported for this window.
+        /// </summary>
+        /// <remarks>
+        /// This doesn't guarantee that context or surface creation will succeed; consult the GLFW documentation for
+        /// more details.
+        /// </remarks>
         public bool IsVulkanSupported => _glfw.VulkanSupported();

```

Comment:
Was there not any documentation to inherit for this method? If not, these doc comments should be moved to the relevant interface in Windowing.Common instead